Feature/add-xpanse-integration-command-get-external-websites (#31978)
demisto/content: Packs/CortexXpanse/ReleaseNotes/1_0_17.md

#### Integrations

##### Cortex Xpanse

> - Updated the Cortex Xpanse integration to add the ***asm-list-external-websites*** command.

Reviewer: ```suggestion
Added the ***asm-list-external-websites*** command to get a list of the external websites assets.
```
